test/functional/core/job_spec.lua

```diff
@@ -1295,6 +1295,27 @@ describe('jobs', function()
       ]])
     end
   end)
+
+  it('uses real pipes for stdin/stdout #35984', function()
+    skip(is_os('win'), 'Not applicable for Windows')
```

Comment:
if it's not applicable I would suggest early `return`, since otherwise it just adds noise to the "SKIPPED" list